lib/cylc/config.py

```diff
@@ -522,6 +522,12 @@ def __init__(self, suite, fpath, template_vars=None,
 
         self.collapsed_families_rc = (
             self.cfg['visualization']['collapsed families'])
+        for fam in self.collapsed_families_rc:
+            if fam not in self.runtime['first-parent descendants'].keys():
```

Comment:
Can leave out the call to `.keys()`, i.e.:

`if fam not in self.runtime['first-parent descendants']:`